### DISCUSSION TITLE
Contributor Call: 26 November 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#3106] [Support]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @lexander2002.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

**Details**
Location: https://discord.freesewing.org (meeting room: 🔊`voice chat`)
Date: 26 November 2022 **5pm** UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/nov-26-2022/5pm).
Or check the bot in Discord! `#meetings`
The meetings will always be at 6pm, Antwerp time.
⚠There have been timezone switches lately, careful!

### Agenda
- TBD

> Notes will be added after the call.

**Comment by darigovresearch** [comment]
Thanks again for the great call, here are two issues that has been raised from some of the discussion in the call

[[bug]: Dead link to Jargon site #3109](https://github.com/freesewing/freesewing/issues/3109)
[[docs]: Add Jargon/Glossary page #3110](https://github.com/freesewing/freesewing/issues/3110)

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
